ChristianChiarulli/LunarVim
# [Feature] Rename lv-config.lua to config.lua

# Description

Rename default user config file from lv-config.lua to config.lua
It still accepts lv-config.lua when config.lua is not found.
Installer creates config.lua as well from now on.

## How Has This Been Tested?

- Run lvim with/without config.lua/lv-config.lua 
- Check autocmd
- Settings option from Dashboard
